versioned/storage/common-tests/src/main/java/org/projectnessie/versioned/storage/commontests/AbstractRepositoryLogicTests.java

```diff
@@ -246,15 +247,21 @@ public void updateRepositoryDescription() throws RetryTimeoutException {
 
     RepositoryDescription updated =
         RepositoryDescription.builder()
-            .defaultBranchName("main")
             .putProperties("updated", "true")
+            // read-only properties, should be ignored
+            .defaultBranchName("main2")
```

Comment:
Heh, exactly this one might be updated. Let's remove "defaultBranchName" from that check.
I'm not sure whether there'll ever be a command to update arbitrary information - rather per attribute.